src/py/aspen/database/models/accessions.py

```diff
@@ -24,23 +26,52 @@ class PublicRepositoryType(enum.Enum):
 )
 
 
-class Accession(idbase):
+class Accession(Entity):
     """A single accession of an entity."""
 
     __tablename__ = "accessions"
     __table_args__ = (UniqueConstraint("repository_type", "public_identifier"),)
+    __mapper_args__ = {"polymorphic_identity": EntityType.PUBLIC_REPOSITORY_SUBMISSION}
 
-    entity_id = Column(
-        Integer,
-        ForeignKey(Entity.id),
-        nullable=False,
-    )
-    entity = relationship(Entity, backref=backref("accessions", uselist=True))
+    entity_id = Column(Integer, ForeignKey(Entity.id), primary_key=True)
 
+    # statistics for the pathogen genome
```

Comment:
is this comment correct? `repository_type` doesn't seem like it would refer to statistics

Reply:
nope, this is wrong.

Reply:
removed.